fluent-bit/fluent-bit-go-loki/kubernetes-example.yaml

```diff
@@ -0,0 +1,129 @@
+kind: ConfigMap
+metadata:
+  name: fluent-bit-config
+apiVersion: v1
+data:
+  fluent-bit.conf: |-
+    [SERVICE]
+        Flush          1
+        Daemon         Off
+        Log_Level      info
+        Parsers_File   parsers.conf
+    [INPUT]
+        Name           tail
+        Tag            kube.*
+        Path           /var/log/containers/*.log
+        Parser         docker
+        DB             /var/log/flb_kube.db
+        Mem_Buf_Limit  5MB
+    [FILTER]
+        Name           kubernetes
+        Match          kube.*
+        Kube_URL       https://kubernetes.default.svc:443
+        Merge_JSON_Log On
+    [Output]
+        Name loki
+        Match *
+        Url ${LOKI_URL}
+        Labels {job="fluent-bit"}
+        RemoveKeys kubernetes,stream
+        LabelMapPath /fluent-bit/etc/labelmap.json
+        LineFormat json
+        LogLevel debug
+  parsers.conf: |-
+    [PARSER]
+        Name        docker
+        Format      json
+        Time_Key    time
+        Time_Format %Y-%m-%dT%H:%M:%S %z
+  labelmap.json: |-
+    {
+        "kubernetes": {
+            "namespace_name": "namespace",
+            "labels": {
+                "component": "component",
+                "tier": "tier"
+            },
+            "host": "host",
+            "container_name": "container",
+            "pod_name": "instance"
+        },
+        "stream": "stream"
+    }
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: fluent-bit
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: fluent-bit-read
+rules:
+- apiGroups: [""]
+  resources:
+  - namespaces
+  - pods
+  verbs: ["get", "list", "watch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: fluent-bit-read
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: fluent-bit-read
+subjects:
+- kind: ServiceAccount
+  name: fluent-bit
+  namespace: logging
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: fluent-bit
+  labels:
+    k8s-app: fluent-bit-logging
+    version: v1
+    kubernetes.io/cluster-service: "true"
+spec:
+  template:
+    metadata:
+      labels:
+        k8s-app: fluent-bit-logging
+        version: v1
+        kubernetes.io/cluster-service: "true"
+    spec:
+      containers:
+      - name: fluent-bit
+        image: grafana/fluent-bit-plugin-loki:latest
+        env:
+          - name:  LOKI_URL
+            value: http://host.docker.internal:3100/api/prom/push
```

Comment:
remove in favor of new path?

Reply:
This file is gone, I've pushed a helm chart, last bit is more documentation for how to run inside ocker/k8s or linux.